app/services/weather_service.rb

```diff
@@ -0,0 +1,18 @@
+class WeatherService
+  def forecast(lat, lon)
+    to_json("/data/2.5/onecall?lat=#{lat}&lon=#{lon}&exclude=minutely&units=imperial")
```

Comment:
excluding minutely data because it is not needed and converting all units to imperial to save the FE lots of conversions